--- kmplot/kmplot/kmplot.desktop.orig	2011-01-19 23:06:31.000000000 +0100
+++ kmplot/kmplot/kmplot.desktop	2011-01-22 02:11:21.229550350 +0100
@@ -134,7 +134,7 @@
 Name[x-test]=xxKmPlotxx
 Name[zh_CN]=KmPlot
 Name[zh_TW]=數學_KmPlot
-Categories=Qt;KDE;Education;Math;
+Categories=Qt;KDE;Education;Science;Math;
 MimeType=application/x-kmplot;
 GenericName=Mathematical Function Plotter
 GenericName[ast]=Trazador de funciones matemátiques
