--- cmake/modules/FindR.cmake.orig	2010-05-05 11:39:33.000000000 +0200
+++ cmake/modules/FindR.cmake	2010-05-28 21:06:46.899757356 +0200
@@ -46,7 +46,7 @@
 
   FIND_LIBRARY(R_R_LIBRARY
     R
-    HINTS ${R_HOME}/lib ${R_SHARED_LIB_DIR} ${R_HOME}/bin )
+    HINTS /usr/lib /usr/lib64 ${R_HOME}/lib ${R_SHARED_LIB_DIR} ${R_HOME}/bin )
   IF(NOT R_R_LIBRARY)
     MESSAGE(STATUS "libR not found. Make sure the location of R was detected correctly, above, and R was compiled with the --enable-shlib option")
   ELSE(NOT R_R_LIBRARY)
