--- step/step/CMakeLists.txt.orig	2010-07-01 00:32:30.880150862 +0200
+++ step/step/CMakeLists.txt	2010-07-01 00:32:49.042969843 +0200
@@ -67,7 +67,8 @@
     ${KDE4_KHTML_LIBS}
     ${KDE4_KNEWSTUFF2_LIBS}
     ${QT_QTOPENGL_LIBRARY}
-    ${qalculate_LIBS})
+    ${qalculate_LIBS}
+    cln)
 
 install(TARGETS step ${INSTALL_TARGETS_DEFAULT_ARGS})
 install(FILES step.kcfg DESTINATION  ${KCFG_INSTALL_DIR})
